### PR TITLE
Clarify installfest flow

### PR DESCRIPTION
**Pull request by amandabeiner**
**Goal** 
- Make the download and Installfest instructions clearer
- Make Cloud9 setup seem like less of an afterthought
- Make sure that the Installfest checklist applies to all environments

**The Problem**
- Students often enter the Installfest documentation from different entry points (ie clicking on Downloads vs Installfest), and get different information depending on their point of entry (ie, they might not land on the "Check your OS" page)
- We don't currently have any resources for setting up the curriculum on Cloud9. TA's might not be familiar with Cloud9, which causes setup delays. This can lead to Cloud9 users feeling overwhelmed, or a sense of failure before they start, and unintentional "machine shaming".

**Changes**
- Add @pcai's  the "Find your OS" tool to the downloads page, in case students land there
- Add a page about how to set up Cloud9, and link to it from the "Downloads" and "Install Environment" pages.
- Make sure the Cloud9 setup instructions link straight to the "Installfest Checklist", and do not prompt users to set up an editor, run vagrant commands, etc
- Modify the Installfest checklist supports all environments

### CODE REVIEW

File: sites/en/downloads/getUserOs.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  // map string fragment to osx version
+  osxVersions: {
+    "10_6": {name: "10.6 Snow Leopard", bitness: "unknown"},
+    "10_7": {name: "10.7 Lion", bitness: "64bit"},
+    "10_8": {name: "10.8 Mountain Lion", bitness: "64bit"},
+    "10_9": {name: "10.9 Mavericks", bitness: "64bit"},
+    "10_10": {name: "10.10 Yosemite", bitness: "64bit", modern: true},
+    "10_11": {name: "10.11 El Capitan", bitness: "64bit", modern: true},
+    "10_12": {name: "10.12 Sierra", bitness: "64bit", modern: true},
+    "10_13": {name: "10.13 High Sierra", bitness: "64bit", modern: true}
```
Comment (edwardloveall) [inline sites/en/downloads/getUserOs.js]: Can we add Mojave to this list?

```js
"10_14": {name: "10.14 Mojave", bitness: "64bit", modern: true}
```

File: sites/en/installfest/checklist.md [hunk 2]
```diff
@@ -24,6 +26,48 @@
 1. Get a setup confirmation sticker from a TA and stick it on
    your laptop where it will be visible when you return tomorrow.
 
+### If you're using Cloud9
+
+1. Log into your Cloud9 console if you're not already there.
+    - Visit (aws.amazon.com/cloud9)[https://aws.amazon.com/cloud9]
```
Comment (edwardloveall) [inline sites/en/installfest/checklist.md]: I think you meant to switch the brackets and parenthesis 🙂

File: sites/en/installfest/checklist.md [hunk 3]
```diff
@@ -24,6 +26,48 @@
...
+    first_number = 1
+    second_number = 2
+    first_number + second_number
+    ```
+6. Type `exit` when you are done with your `irb` session. That's it!
+
+### If you're using native install
+
+1. Open the terminal
+  - **On Macs:** In Finder, start Applications > Utilities > Terminal, or find the Terminal application through Spotlight (click the magnifying glass in the top right of the screen and start typing 'Terminal')
```
Comment (edwardloveall) [inline sites/en/installfest/checklist.md]: Nitpick but I think `On Mac` reads better or `On macOS`. How do you feel about that?

**Review by edwardloveall — APPROVED** [review]
This is great! Approved as long as you at least update the os checking script to include 10.14. Couple other optional changes also,

File: sites/en/installfest/checklist.md (hunk 3 above)
Comment (amandabeiner) [reply sites/en/installfest/checklist.md]: That makes sense to me! This was just copy-pasta'd from above, so I will change it in both places

File: sites/en/installfest/cloud9.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+7. Select the "Basic Plan" from the next screen. This plan is free.
+
+8. Click "Continue to the Console" to access your new Amazon Web Services account!
+
+### Create a new project
+
+1. Click on "Services" in the top left navigation menu.
+
+2. Select "Could9" under "Developer Tools", or search for and select Cloud9 in the search bar.
```
Comment (raadler) [inline sites/en/installfest/cloud9.md]: That first cloud9 is misspelled.

**Review by raadler — APPROVED** [review]
Other than the one typo I found, it looks good to me.

**Pull request merged**